R/ernm-models.R

```diff
@@ -82,6 +84,12 @@ MissingErnmModel <- function(observedSampler, unobservedSampler, ...){
 FullErnmModel <- function(sampler, logLik, ...){
 	
 	sampler <- sampler
+	model <- sampler$getModel()
+	if(class(model)[1] == 'Rcpp_UndirectedModel' | class(model)[1] == 'Rcpp_DirectedModel'){
+	    tapered <- FALSE
+	}else{
```

Comment:
Flip this condition. Check if the underlying Rcpp model is a tapering model and if so set tapering <- TRUE. I'm not sure if every class that isn't UndirectedModel or DirectedModel will forever and always be a tapered model. Also use `inherits` not `class(...)[1] == ...`.